packages/office-addin-debugging/src/cli.ts

```diff
@@ -4,7 +4,7 @@
 // Licensed under the MIT license.
 
 import * as commander from "commander";
-import { logErrorMessage } from "office-addin-cli";
+import { logErrorMessage } from "office-addin-usage-data";
```

Comment:
shouldn't there be a matching package.json change to import usage-data?

Reply:
I already had it imported

Reply:
But you can now remove cli from package.json

Reply:
This one still had a call for parseNumber and for getPackageJsonScript